tests/scene/test_node.h

```diff
@@ -343,6 +352,24 @@ TEST_CASE("[SceneTree][Node] Testing node operations with a more complex simple
 		CHECK_EQ(Object::cast_to<Node>(children[1]), node1);
 	}
 
+	SUBCASE("Nodes should not be possible to find") {
```

Comment:
Which nodes? Maybe "Invalid nodes" or "Nodes with invalid name"...

Reply:
It should be testing for both invalid names and/or types.

Reply:
How about `Nodes should not be possible to find by name and/or type`

Reply:
> How about `Nodes should not be possible to find by name and/or type`

Still missing the essential part "invalid" 😉 

Something along "Nodes cannot be found if type or name is invalid", straight and simple.

Reply:
> Something along "Nodes cannot be found if type or name is invalid", straight and simple.

I like that phrasing. I am now using that.